godtools/Views/BaseTractView.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  BaseTractView.swift
+//  godtools
+//
+//  Created by Ryan Carlson on 4/26/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SWXMLHash
+
+class BaseTractView: UIView {
+    
+    var data: XMLIndexer?
+    
+    override func awakeFromNib() {
```

Comment:
I don't think we need the override method if we're not doing anything in it. It helps keep the codebase cleaner.